packages/babel-register/test/cache.js

```diff
@@ -73,5 +73,42 @@ describe("@babel/register - caching", () => {
       expect(fs.existsSync(testCacheFilename)).toBe(true);
       expect(get()).toEqual({});
     });
+
+    it("should create the cache after load", cb => {
+      load();
+
+      process.nextTick(() => {
+        process.nextTick(() => {
```

Comment:
[Q]: Why isn't one `nextTick` enough?

Reply:
[A]: I thought the execution order of `next tick` tasks is not defined. Apparently I am wrong after reading https://nodejs.org/docs/latest/api/process.html#process_process_nexttick_callback_args, thank you for reminding me!